src/main/java/com/internet/shop/models/Order.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.shop.models;
+
+import java.util.List;
+
+public class Order {
+    private long id;
+    private List<Product> products;
+    private long idUser;
+
+    public Order(long id, List<Product> products, long idUser) {
+        this.id = id;
+        this.products = products;
+        this.idUser = idUser;
```

Comment:
let's not pass list to constructor but just initialize it here

Reply:
fixed it